.github/workflows/check-dist.yml

```
@@ -21,10 +21,10 @@ jobs:
       - name: Enable corepack
         run: corepack enable
 
-      - name: Set Node.js 16.x
-        uses: actions/setup-node@5e21ff4d9bc1a8cf6de233a3057d20ec6b3fb69d # v3.8.1
+      - name: Set Node.js 20
```

```suggestion
      - name: Set Node.js
```
action内でバージョンへの関心がなくなったので消して良いと思います

---

@nonsugarless 確かに漏れてたので修正しました 👍